app/jinja_filters.py

```diff
@@ -126,3 +133,8 @@ def start_end_date_check():
 @blueprint.app_context_processor
 def conditional_dates_check():
     return dict(format_conditional_date=format_conditional_date)
+
+
+@blueprint.app_context_processor
+def unit_check():
```

Comment:
I think it would be clearer for this to be called `format_unit_processor`, I'm not sure why the others are called `_check`